src/views/v2/list/nav/next-disabled.php

```diff
@@ -11,12 +11,25 @@
  *
  * @var string $link The URL to the next page, if any, or an empty string.
  *
- * @version 4.9.4
+ * @version TBD
  *
  */
 ?>
 <li class="tribe-events-c-nav__list-item tribe-events-c-nav__list-item--next">
 	<button class="tribe-events-c-nav__next tribe-common-b2" disabled>
-		<?php echo esc_html( sprintf( __( 'Next %s', 'the-events-calendar' ), tribe_get_event_label_plural() ) ); ?>
+		<?php
+			echo sprintf(
+				wp_kses(
+					/* translators: %s: Event (plural or singular). */
+					__( 'Next<span class="tribe-events-c-nav__next--label-plural"> %s</span>', 'the-events-calendar' ),
+					array(
+						'span' => array(
+							'class' => array(),
+						),
+					)
+				),
+				tribe_get_event_label_plural()
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: e91341f1326909816cc9539cf77a8016</sub>